nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -140,50 +140,58 @@ $(CsWinRTFilters)
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Prerelease.targets" Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Prerelease.targets')"/>
 
   <!-- Copy Authored winmd to output folder -->
-  <Target Name="CsWinRTPlaceWinMDInOutputFolder"
-          Condition="'$(CsWinRTComponent)' == 'true'" 
-          BeforeTargets="AfterBuild">
+  <Target Name="CsWinRTPlaceWinMDInOutputFolder" Condition="'$(CsWinRTComponent)' == 'true'" BeforeTargets="AfterBuild">
     <Copy SourceFiles="$(GeneratedFilesDir)\$(AssemblyName).winmd" DestinationFolder="$(TargetDir)" UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
   </Target>
 
-  <!-- When an authored component packs (nuget), add the necessary hosting assets to the package -->
-  <Target Name="CsWinRTIncludeHostDlls"  
-          Condition="'$(CsWinRTComponent)' == 'true'" 
-          BeforeTargets="GenerateNuspec" 
-          Outputs="@(_PackageFiles)">
+  <!-- When an authored component makes a nupkg, add the necessary hosting assets to the package -->
+  <Target Name="CsWinRTIncludeHostDlls"  Condition="'$(CsWinRTComponent)' == 'true'" BeforeTargets="AfterBuild" Outputs="@(Content)">
     <ItemGroup>
-      <_PackageFiles Include="$(TargetDir)$(AssemblyName).winmd"
-                     PackagePath="winmd\"/>
+      <Content Include="$(TargetDir)$(AssemblyName).winmd"
+               Pack="true"
+               PackageCopyToOutput="true"
+               PackagePath="winmd" />
       
       <!-- The authored component gets the WinRT.Runtime and NET.SDK dlls from ReferenceCopyLocalPaths 
-            (because CopyLocalLockFileAssemblies == true); $(TargetPath) has the assembly (e.g. PackageId.dll) -->
-      <_PackageFiles Include="$(CsWinRTPath)lib\net5.0\WinRT.Host.Shim.dll;@(ReferenceCopyLocalPaths);$(TargetPath)"
-                     PackagePath="lib\native"/>
+            (because CopyLocalLockFileAssemblies == true) and $(TargetPath) has the assembly (e.g. PackageId.dll) -->
+      <Content Include="$(CsWinRTPath)lib\net5.0\WinRT.Host.Shim.dll;@(ReferenceCopyLocalPaths);$(TargetPath)"
+               Pack="true"
+               PackageCopyToOutput="true"
+               PackagePath="native" />
```

Comment:
Are we intentionally moving these out of the lib folder?

Reply:
I figured it would be nicer to have a shorter path, and don't think the lib folder is that important for our purposes. Do you?

Reply:
I restored the lib folder.